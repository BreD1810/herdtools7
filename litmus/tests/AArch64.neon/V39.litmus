AArch64 V39
(* Test ADD (vector, scalar variant) *)
{
int64_t x=1; int64_t y=2; int64_t z;
0:X0=x; 0:X1=y; 0:X2=z;
}

P0;
  LD1 {V0.D}[0],[X0]    ;
  LD1 {V1.D}[0],[X1]    ;
  ADD D2,D0,D1          ;
  STR D2,[X2]           ;

forall (z=3)
