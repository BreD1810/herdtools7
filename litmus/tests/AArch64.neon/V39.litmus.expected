Test V39 Required
Histogram (1 states)
1000000:>z=3;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (z=3) is validated
Hash=34b382984f80d17d1950438f9b681e27
Observation V39 Always 1000000 0
